packages/entities/entities-plugins/src/components/PluginIcon.vue

```diff
@@ -31,11 +30,27 @@ defineProps({
   },
 })
 
+const defaultIcon = ref('')
+const src = ref('')
```

Comment:
suggestion: can we rename this different than the native attribute? `imgSrc` or something?